api_payments.yaml

```diff
@@ -0,0 +1,286 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Proxy PagoPA
+  description: Proxy for PagoPA payment services
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+paths:
+  "/payment/check":
+    x-swagger-router-controller: PaymentController
+    get:
+      operationId: checkPaymentToPagoPa
+      summary: Check Payment
+      description: Retrieve information about a payment (qrcode)
+      parameters:
+        - in: body
+          name: body
+          schema:
+            $ref: "#/definitions/PaymentsCheckRequest"
+          x-examples:
+            application/json:
+              codiceIdRPT:
+                CF: "TMMEXQ60A10Y526X"
+                AuxDigit: 1
+                CodIUV: 123456789012345
+                CodStazPA: 56
+      responses:
+        '200':
+          description: Payment information retrieved
+          schema:
+            "$ref": "#/definitions/PaymentsCheckResponse"
+          examples:
+            application/json:
+              importoSingoloVersamento: 200,
+              codiceContestoPagamento: "ABC123"
+        "400":
+          description: Input provided or pagoPa response are invalid
```

Comment:
we must choose a way to write pagopa into specs and code. I vote for `PagoPA`

Reply:
I agree. Most referencies use PagoPA. I'll update different values